### ISSUE TITLE
The description of `ServiceWorker`'s getter is not comprehensive enough

### ISSUE THREAD

**Issue by skyclouds2001** [mdn/content#29541]
### MDN URL

https://developer.mozilla.org/en-US/docs/Web/API/ServiceWorker

### What information was incorrect, unhelpful, or incomplete?

```md
A ServiceWorker object is available in the `ServiceWorkerRegistration.active` property, and the `ServiceWorkerContainer.controller` property — this is a service worker that has been activated and is controlling the page (the service worker has been successfully registered, and the controlled page has been reloaded.)
```

The getter description of ServiceWorker only mentioned `ServiceWorkerRegistration.active` and `ServiceWorkerContainer.controller`

However, ServiceWorker may get via `ServiceWorkerGlobalScope.serviceWorker` and `ServiceWorkerRegistration.installing` also `ServiceWorkerRegistration.waiting`

### What did you expect to see?

Update this description to fully reflect ServiceWorker's getter

### Do you have any supporting links, references, or citations?

**Added yet**

https://developer.mozilla.org/en-US/docs/Web/API/ServiceWorkerRegistration/active

https://developer.mozilla.org/en-US/docs/Web/API/ServiceWorkerContainer/controller

**Need Add**

See #29123 and #29461 

https://developer.mozilla.org/en-US/docs/Web/API/ServiceWorkerRegistration/installing

https://developer.mozilla.org/en-US/docs/Web/API/ServiceWorkerRegistration/waiting

**Related documentation**

https://w3c.github.io/ServiceWorker/#serviceworker-interface

https://w3c.github.io/ServiceWorker/#serviceworkerregistration-interface

https://w3c.github.io/ServiceWorker/#serviceworkercontainer-interface

### MDN metadata

<details>
<summary>Page report details</summary>

* Folder: `en-us/web/api/serviceworker`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/API/ServiceWorker
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/api/serviceworker/index.md
* Last commit: https://github.com/mdn/content/commit/33fe6977c5dcbf4d5d89ed1faa5559a778936b65
* Document last modified: 2023-05-15T05:31:24.000Z

</details>

**Comment by Lymah123** [comment]
Hi @skyclouds2001 and @Josh-Cena , can I work on the issue?

**Comment by skyclouds2001** [comment]
> Hi @skyclouds2001 and @Josh-Cena , can I work on the issue?

I'm Okay

**Comment by Lymah123** [comment]
Please assign me to the issue.

**Comment by Josh-Cena** [comment]
We don't assign folks—you can send a PR right away

**Comment by Lymah123** [comment]
Okay. I'm on it then!

**Comment by skyclouds2001** [comment]
@Lymah123 will you fix it, or I will make the changes in #29461

**Comment by Lymah123** [comment]
@skyclouds2001! I will work on it.
